new_2.txt

```diff
@@ -0,0 +1 @@
+Hi my dear friends.
```

Comment:
nope, you can't commit to the root of repo. You can commit only to your personal directory in "personal/" dir.